.storybook/preview.jsx

```diff
@@ -31,19 +31,6 @@ export const decorators = [StylesWrapperDecorator]
 
 export const parameters = {
   actions: { argTypesRegex: '^on[A-Z].*' },
-  backgrounds: {
-    default: 'black',
-    values: [
-      {
-        name: 'black',
-        value: '#000000',
-      },
-      {
-        name: 'gray',
-        value: '#272D33',
-      },
-    ],
-  },
```

Comment:
why we removing the backgrounds aren't those the same bgs as our app?

Reply:
Storybook was warning me about this option being deprecated. Removing it didn't have any effect, the background for canvas is still black, because of our `GlobalStyle`